packages/cli/README.md

```diff
@@ -1245,7 +1249,127 @@ ALIASES
 
 _See code: [dist/commands/locations/rooms/update.ts](https://github.com/SmartThingsCommunity/smartthings-cli/blob/v0.0.0/dist/commands/locations/rooms/update.ts)_
 
+<<<<<<< HEAD
```

Comment:
Some conflict markers got left in here...I would recommend just regenerating the README with a build.